.nais/preprod.yaml

```diff
@@ -32,8 +32,6 @@ spec:
   ingresses:
     - https://app-q0.adeo.no/modiapersonoversikt-innstillinger
     - https://app-q1.adeo.no/modiapersonoversikt-innstillinger
-    - https://modia-q0.dev.adeo.no/modiapersonoversikt-innstillinger
-    - https://modia-q1.dev.adeo.no/modiapersonoversikt-innstillinger
```

Comment:
Hvorfor sletter du url for dev domain ?

Reply:
`modia.adeo.no` og `modia-qX.dev.adeo.no` ble testet ut før vi fant ut at bytte av cookie-navn var greieste løsningen.
Siden domene dermed ikke lenger er nødvendig så fjerner vi det.